src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java

```diff
@@ -29,16 +29,16 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 import org.xmlunit.matchers.CompareMatcher;
 
 /**
  * Test for {@link PrimaryOutput}.
  * @since 0.10
- * @todo #241:30min Extend this test: add test methods to check `file()`, `maid()` and `tag()`
- *  methods of `PrimaryOutput` and add method to check generated primary.xml for libdeflt test
- *  rpm package.
+ * @todo #277:30min Extend this test: add test methods to check `maid()` method of
+ *  PrimaryOutput`
```

Comment:
@paulodamaso test for `maid()` method should only check that `maid()` returns correct instance of `XmlMaid`, maybe you could add here please? I do not think that it should be done in separate PR...